src/shared/sam/debugger/awsSamDebugger.ts

```diff
@@ -105,6 +115,12 @@ export class SamDebugConfigProvider implements vscode.DebugConfigurationProvider
         if (token?.isCancellationRequested) {
             return undefined
         }
+        /**
+         * XXX: Temporary magic field for testing.
+         * 1. Disables the EXECUTE phase.
+         * 2. Returns the config.
+         */
+        const configOnly = !!(config as any).configOnly
```

Comment:
Could this be named more blatantly, to show that it is for internal purposes, and that it is intended to not run the invoke? For example `__disableInvoke`

Reply:
made that change, but this can be removed pretty quickly, there's nothing stopping us except that this PR is transitional.